core/src/test/scala/org/apache/spark/sql/PartitionTableSuite.scala

```diff
@@ -278,65 +396,52 @@ class PartitionTableSuite extends BaseTiSparkSuite {
-//    judge("select yearweek(d) from part_fn")
-//    judge("select weekday(d) from part_fn")
-//    judge("select to_days(d) from part_fn")
-//    judge("select microsecond(d) from part_fn")
-//    judge("select time_to_sec(d) from part_fn")
-//  }
-
-//  test("partition expr can be parsed by sparkSQLParser") {
-//    assert(spark.sql("select Abs(1)").count == 1)
-//    assert(spark.sql("select Ceiling(1)").count == 1)
-//    assert(spark.sql("SELECT datediff('2009-07-31', '2009-07-30')").count == 1)
-//    assert(spark.sql("SELECT day('2009-07-30')").count() == 1)
-//    assert(spark.sql("SELECT dayofmonth('2009-07-30')").count() == 1)
-//    assert(spark.sql("SELECT dayofweek('2009-07-30')").count() == 1)
-//    assert(spark.sql("SELECT dayofyear('2016-04-09')").count() == 1)
-//    assert(spark.sql("SELECT floor(-0.1)").count() == 1)
-//    assert(spark.sql("SELECT hour('2009-07-30 12:58:59')").count() == 1)
-//    assert(spark.sql("SELECT minute('2009-07-30 12:58:59')").count() == 1)
-//    assert(spark.sql("SELECT mod(2, 1)").count() == 1)
-//    assert(spark.sql("SELECT month('2016-07-30')").count() == 1)
-//    assert(spark.sql("SELECT quarter('2016-08-31')").count() == 1)
-//    assert(spark.sql("SELECT second('2009-07-30 12:58:59')").count() == 1)
-//    assert(spark.sql("SELECT unix_timestamp('2016-04-08', 'yyyy-MM-dd')").count() == 1)
-//    assert(spark.sql("SELECT unix_timestamp()").count() == 1)
-//    assert(spark.sql("SELECT year('2009-07-30 12:58:59')").count() == 1)
-//    //     extract is not supported
-//    // https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_extract
-//    judge("select time_to_sec('12:30:49')")
-//    judge("select time_to_sec('2018-10-10 12:30:49')")
-//    judge("select to_days('2018-10-10')")
-//    judge("select to_days('2018-10-10 12:30:49')")
-//    judge("select to_seconds('2018-10-01 12:34:59')")
-//    judge("select to_seconds('2018-10-01')")
-//    judge("select microsecond('2018-10-01 12:34:59')")
-//    judge("select weekday('2018-09-12')")
-//    judge("select weekday('2018-01-01')")
-//    judge("select yearweek('1992-01-01')")
-//    judge("select yearweek('1992-12-31')")
-//    judge("select yearweek('1992-01-01', 1)")
-//    judge("select yearweek('1992-12-31', 1)")
-//    judge("select yearweek('1992-01-01', 2)")
-//    judge("select yearweek('1992-12-31', 2)")
-//    judge("select yearweek('1992-01-01', 3)")
-//    judge("select yearweek('1992-12-31', 3)")
-//    judge("select yearweek('1992-01-01', 4)")
-//    judge("select yearweek('1992-12-31', 4)")
-//    judge("select yearweek('1992-01-01', 5)")
-//    judge("select yearweek('1992-12-31', 5)")
-//    judge("select yearweek('1992-01-01', 6)")
-//    judge("select yearweek('1992-12-31', 6)")
-//    judge("select yearweek('1992-01-01', 7)")
-//    judge("select yearweek('1992-12-31', 7)")
-//  }
+  ignore("partition expr can be parsed by sparkSQLParser") {
+    assert(spark.sql("select Abs(1)").count == 1)
```

Comment:
Likely we need to add prefix to distinguish btw spark's udf and ours.

Reply:
My bad. It can be deleted since we are not using spark's parser.